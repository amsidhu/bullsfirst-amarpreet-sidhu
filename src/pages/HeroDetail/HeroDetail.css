.detail-container {
  position: absolute;
  width: 432px;
  top: 316px;
  left: 24px;
}

.text-default {
  font-family: Inter;
  font-style: normal;
  font-weight: normal;
  font-size: 16px;
  line-height: 26px;
}

.text-default-italic {
  font-family: Inter;
  font-style: italic;
  font-weight: normal;
  font-size: 16px;
  line-height: 26px;
}

.text-sm-italic {
  font-family: Inter;
  font-style: italic;
  font-weight: normal;
  font-size: 13px;
  line-height: 22px;
}
