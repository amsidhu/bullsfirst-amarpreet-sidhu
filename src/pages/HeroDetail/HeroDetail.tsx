import React from 'react';
import './HeroDetail.css';

export const HeroDetail = () => {
  return (
    <div className="detail-container">
      <p className="text-default">Every account comes with:</p>
      <ul>
        <li className="text-default">0-second execution</li>
        <li className="text-default">Portfolio insurance</li>
        <li className="text-default">1 popsicle</li>
      </ul>
      <p className="text-default">We love to trade, and we hope you do too! </p>
      <p className="text-default">
        Now you can express that love with the latest trading platform from
        Bullsfirst. We guarantee a 0-second trade execution, and will give you a
        free popsicle if your trade doesn’t go through in this time period.
      </p>
      <p className="text-default-italic">
        On a side-note, we just bought a big freezer and 10,000 popsicles.
        Thankfully, we’ve hedged all popsicle expenses for the next two years!
        (Commodity experts say popsicles are on the rise).
      </p>
      <p className="text-sm-italic">
        This is a demo application. All data displayed is fictitious.
      </p>
    </div>
  );
};
