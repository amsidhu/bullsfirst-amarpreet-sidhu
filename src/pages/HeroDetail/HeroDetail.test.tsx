import React from 'react';
import { render } from '../../test/test-utils';
import { HeroDetail } from './HeroDetail';

describe('<HeroDetail />', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<HeroDetail />);
    expect(asFragment()).toMatchSnapshot();
  });
});
