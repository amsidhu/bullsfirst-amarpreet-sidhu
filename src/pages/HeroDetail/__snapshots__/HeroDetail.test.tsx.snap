// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeroDetail /> renders correctly 1`] = `
<DocumentFragment>
  <div
    class="detail-container"
  >
    <p
      class="text-default"
    >
      Every account comes with:
    </p>
    <ul>
      <li
        class="text-default"
      >
        0-second execution
      </li>
      <li
        class="text-default"
      >
        Portfolio insurance
      </li>
      <li
        class="text-default"
      >
        1 popsicle
      </li>
    </ul>
    <p
      class="text-default"
    >
      We love to trade, and we hope you do too! 
    </p>
    <p
      class="text-default"
    >
      Now you can express that love with the latest trading platform from Bullsfirst. We guarantee a 0-second trade execution, and will give you a free popsicle if your trade doesn’t go through in this time period.
    </p>
    <p
      class="text-default-italic"
    >
      On a side-note, we just bought a big freezer and 10,000 popsicles. Thankfully, we’ve hedged all popsicle expenses for the next two years! (Commodity experts say popsicles are on the rise).
    </p>
    <p
      class="text-sm-italic"
    >
      This is a demo application. All data displayed is fictitious.
    </p>
  </div>
</DocumentFragment>
`;
